contentcuration/contentcuration/serializers.py

```diff
@@ -76,53 +77,20 @@ def update(self, instance, validated_data):
         user = self.context['request'].user
         import ipdb
         with transaction.atomic():
+        # Get files that have the same contentnode, preset, and language as the files that are now attached to this node
             for item in validated_data:
-                ipdb.set_trace()
-                # item.update({
-                #     # 'preset_id': item['preset']['id'],
-                #     'language_id': item.get('language')['id'] if item.get('language') else None
-                # })
-
-                # # User should not be able to change files without a display
-                # if item['preset']['display']:
-                #     if 'id' in item:
-                #         update_files[item['id']] = item
-                #     else:
-                #         # create new nodes
-                #         ret.append(File.objects.create(**item))
-                # item.pop('preset', None)
-                # item.pop('language', None)
-
-        files_to_delete = []
-        nodes_to_parse = []
-        current_files = [f['id'] for f in validated_data]
+                file_obj = File.objects.get(pk=item['id'])
```

Comment:
Removes some dead code that was interfering with file uploads/file metadata editing.